content/boilerplates/notes/1.2.1.md

```diff
@@ -0,0 +1,16 @@
+## Bug fixes and minor enhancements
+
+- Fix duplicate CRD being generated in the install ([Issue 14976](https://github.com/istio/istio/issues/14976))
+- Allow users to disable istio default retries by setting retries to 0 ([Issue 14900](https://github.com/istio/istio/issues/14900))
+- Improve citadel log ([Pull Request 15032](https://github.com/istio/istio/pull/15032))
+- Add citadel replicaCount flag back and workloadCertTtl to support multi-citadel and short cert TTL for perf test ([Pull Request 15066](https://github.com/istio/istio/pull/15066))
+- Fix Mixer CRD store issue ([Issue 14841](https://github.com/istio/istio/issues/14841))
+- Add HTTP/1.0 support to gateway configuration generation ([Pull Request 15062](https://github.com/istio/istio/pull/15062))
+- Redis filter (this feature is guarded with the feature flag PILOT_ENABLE_REDIS_FILTER, disabled by default):
+    - Enable redis filter ([Pull Request 15049](https://github.com/istio/istio/pull/15049))
```

Comment:
would be good to specify how to enable/disable this?  even it is pointing to a doc to set this or any ennv var/feature flag.

Reply:
I believe this flag is setup through the same mechanism than those: https://istio.io/docs/reference/commands/istio_ca/#envvars

Reply:
```suggestion
```